interface/practice/ins_search.php

```diff
@@ -168,13 +168,13 @@ function validate(f) {
    $ins_id = generate_id();
 
    sqlInsert("INSERT INTO insurance_companies ( " .
-    "id, name, attn, cms_id, freeb_type, x12_receiver_id, x12_default_partner_id " .
+    "id, name, attn, cms_id, ins_co_type, x12_receiver_id, x12_default_partner_id " .
```

Comment:
as an aside, here's where it is used correctly:
``````
grep -Ri "ins_type_code" /var/www/openemr/
/var/www/openemr/sql/4_2_2-to-4_3_1_upgrade.sql:#IfMissingColumn insurance_companies ins_type_code
/var/www/openemr/sql/4_2_2-to-4_3_1_upgrade.sql:ALTER TABLE `insurance_companies` CHANGE `freeb_type` `ins_type_code` tinyint(2) Default NULL;
/var/www/openemr/sql/database.sql:  `ins_type_code` tinyint(2) default NULL,
/var/www/openemr/templates/insurance_companies/general_edit.html:		{html_options name="ins_type_code" options=$insurancecompany->ins_type_code_array selected=$insurancecompany->get_ins_type_code()}
grep: /var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%76: Permission denied
grep: /var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%1E: Permission denied
/var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%FF^FF0^FF06E40D%%general_edit.html.php:		<?php echo smarty_function_html_options(array('name' => 'ins_type_code','options' => $this->_tpl_vars['insurancecompany']->ins_type_code_array,'selected' => $this->_tpl_vars['insurancecompany']->get_ins_type_code()), $this);?>
grep: /var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%12: Permission denied
grep: /var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%61: Permission denied
grep: /var/www/openemr/interface/main/calendar/modules/PostCalendar/pntemplates/compiled/%%5B: Permission denied
/var/www/openemr/library/gen_hcfa_1500.inc.php:  // claimTypeRaw() gets the integer value from insurance_companies.ins_type_code.
/var/www/openemr/library/gen_hcfa_1500.inc.php:  // Previous version of this code called claimType() which maps ins_type_code to
/var/www/openemr/library/classes/InsuranceCompany.class.php:	var $ins_type_code;
/var/www/openemr/library/classes/InsuranceCompany.class.php:	var $ins_type_code_array = array('','Other HCFA'
/var/www/openemr/library/classes/InsuranceCompany.class.php:	function set_ins_type_code($type) {
/var/www/openemr/library/classes/InsuranceCompany.class.php:		$this->ins_type_code = $type;
/var/www/openemr/library/classes/InsuranceCompany.class.php:	function get_ins_type_code() {
/var/www/openemr/library/classes/InsuranceCompany.class.php:		return $this->ins_type_code;
/var/www/openemr/library/classes/InsuranceCompany.class.php:	function get_ins_type_code_display() {
/var/www/openemr/library/classes/InsuranceCompany.class.php:		return $this->ins_type_code_array[$this->ins_type_code];
/var/www/openemr/library/classes/InsuranceCompany.class.php:		return $this->ins_claim_type_array[$this->ins_type_code];
/var/www/openemr/library/Claim.class.php:    return $this->payers[$ins]['object']->get_ins_type_code();
/var/www/openemr/custom/export_labworks.php:  "c1.name AS name1, c1.ins_type_code AS instype1, " .
/var/www/openemr/custom/export_labworks.php:  "c2.name AS name2, c2.ins_type_code AS instype2, " .